internal/concierge/impersonator/impersonator_test.go

```diff
@@ -55,8 +55,10 @@ import (
 	"go.pinniped.dev/internal/httputil/roundtripper"
 	"go.pinniped.dev/internal/kubeclient"
 	"go.pinniped.dev/internal/testutil/tlsserver"
+	"go.pinniped.dev/internal/tokenclient"
 )
 
+// TODO: add a test without a token?
```

Comment:
What about this TODO?

Reply:
IIRC it was extremely difficult to make a test without a token because of the way the tests are constructed.